cypress/integration/buy-tshirt.cy.ts

```diff
@@ -0,0 +1,42 @@
+import {MenuContentPage} from "../page/menu-content.page";
+import {ProductsListPage} from "../page/products-list.page";
+import {ShopingCartPage} from "../page/shoping-cart.page";
+import {SignInPage} from "../page/login.page";
+import {AddressPage} from "../page/address-step.page";
+import {ShippingPage} from "../page/shipping-step.page";
+import {PaymentPage} from "../page/payment-step.page";
```

Comment:
this can be simplified:
```suggestion
import {
  MenuContentPage,
  ProductListPage,
  ShoppingCartPage,
  LoginPage,
  AddressStepPage,
  ShippingStepPage,
  PaymentStepPage,
} from "../page/index";
```